packages/console-extension/src/index.ts

```diff
@@ -190,7 +191,8 @@ async function activateConsole(
               args: { isLauncher: true, kernelPreference: { name } },
               category: 'Console',
               rank,
-              kernelIconUrl
+              kernelIconUrl,
+              metadata: spec.metadata
```

Comment:
It feels like we should namespace the kernel spec metadata so items can have other metadata too. For example, `metadata: {kernel: spec.metadata}` or something.

Reply:
Hm. That's a good point. 👍

Reply:
That's fair, I'll make the change.